### DISCUSSION TITLE
v2.8.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#2359] [General]
## Changes
- New interface for adding websites to teams
<img src="https://github.com/umami-software/umami/assets/477975/e72feafb-7a54-4ea9-9920-938c4f27fad6" width="600" />

- Lots of responsive CSS updates for mobile
- Added additional validations to all API routes

## Fixes
- Fixed websites dropdown not showing all websites
- Fixed issue with login sometimes redirecting incorrectly
- Fixed parsing of pathname in tracker

A huge thanks to all the contributors on this release! @juangacovas @mho22 @RavelloH @bilguun0203 @aldo-roman @PeterDaveHello 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v2.8.0'>v2.8.0</a>.</em>

**Comment by han1202qd** [comment]
Has the shared link been fixed? It was always showing blank.
